R/test-server.R

```diff
@@ -50,7 +71,9 @@ isModuleServer <- function(x) {
 #'   # Any additional arguments, below, are passed along to the module.
 #' }, multiplier = 2)
 #' @export
-testServer <- function(app = ".", expr, ...) {
```

Comment:
I think we can keep `app = "."` and automatically search up from there, similar to how the functions in devtools and usethis work.

Reply:
While `app` now defaults to `findEnclosingApp(".")` it sounds to me like the behavior is that which you describe, i.e. similar to devtools etc.

Are you suggesting an arrangement more like the following?

```
testServer <- function(app = ".", expr, ...) {
  app <- if (is.character(app)) findEnclosingApp(app) else app
}
```
The complication relative to devtools here is that `app` might be an appobj.

Reply:
Oops, I get it now, if we default with `findEnclosingApp()` than user-supplied dirs aren't traversed. Will compute.